independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -165,6 +188,14 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         return QuarkusCommandOutcome.success();
     }
 
+    private static Collection<AppArtifactCoords> getExtensionsToAdd(ExtensionInstallPlan extensionInstallPlan) {
```

Comment:
That's okay but I think we should fix where the real problem is. The way I see is that there are another two solutions to this problem:

- change in `DefaultExtensionRegistry` to not set the version in managed extensions;
- change the `create()` method to accept an `ExtensionInstallPlan` instead of a collection of `AppArtifactCoords` and fix how dependencies are added (eg. Platforms chosen and if the extension is managed or not)

The second option seems more appealing, but it would require more work than option 1. WDYT?

Reply:
Agree. The second option seems the best solution here, although I think that I will need help to understand the big picture.